include/Util/BasicTypes.h

```diff
@@ -78,7 +80,12 @@ typedef llvm::Function Function;
 typedef llvm::BasicBlock BasicBlock;
 typedef llvm::Value Value;
 typedef llvm::Instruction Instruction;
+#if LLVM_VERSION_MAJOR >= 11
+//typedef llvm::AbstractCallSite CallSite;
+typedef llvm::CallBase* CallSite;
```

Comment:
It is a bit change to define a pointer type to a non-pointer one. SVF users might have to use `->` for field access via a non-pointer type. Could we write a wrapper similar to https://github.com/llvm-mirror/llvm/blob/master/include/llvm/IR/CallSite.h#L672? or any better idea?

Reply:
Ok, I will try it later.

Reply:
Yes, that would be good.